qa/tasks/cephfs/test_volumes.py

```diff
@@ -8165,6 +8165,56 @@ def test_unparseable_metafile_on_legacy_to_v1_upgrade(self):
         # remove group
         self._fs_cmd("subvolumegroup", "rm", self.volname, group)
 
+    def test_dangling_symlink(self):
+        """
+        Tests for the presence of any dangling symlink, if yes,
```

Comment:
nit: lets add the test name at the beginning of the comment. Teuthology picks the first line of the comment while logging the test as failed or passed. It will be useful in grepping.